File: Task Manager/src/CreateIssueCommand.java
public class CreateIssueCommand implements Command{
    public Issue issue;
    public IssueRepository issueRepository;

    public CreateIssueCommand(Issue issue, IssueRepository issueRepository) {
        this.issue = issue;
        this.issueRepository = issueRepository;
    }

    @Override
    public void execute(){
        issueRepository.addIssue(issue);
    }

    @Override
    public void undo(){
        issueRepository.removeIssue(issue);
    }



}

File: TaskManager/MyTaskManager/CreateIssueCommand.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MyTaskManager
{
    public class CreateIssueCommand : Command
    {
        public IssueRepository repository;
        public Issue issue;

        public CreateIssueCommand(IssueRepository repository, string title, string description, Priority priority)
        {
            this.repository = repository;
            int issueId = repository.GetNextIssueId();
            this.issue = new Issue(issueId, title, description, priority);
            this.Type = CommandType.Create;
            this.Description = $"Created JIssueID: {issueId}, Title: {title}";
        }

        public override void Execute()
        {
            repository.AddIssue(issue);
        }

        public override void Undo()
        {
            var repositoryIssues = repository.GetAllIssues();
            repositoryIssues.Remove(issue.Id);
        }
    }
}
